onmt/transforms/tokenize.py

```diff
@@ -14,11 +14,16 @@ def __init__(self, opts):
 
     @classmethod
     def add_options(cls, parser):
-        """Avalilable options relate to Subword."""
+        """Available options relate to Subword."""
         # Sharing options among `TokenizerTransform`s, same name conflict in
         # this scope will be resolved by remove previous occurrence in parser
         group = parser.add_argument_group(
-            'Transform/Subword/Common', conflict_handler='resolve')
+            'Transform/Subword/Common', conflict_handler='resolve',
+            description=".. Attention:: Common options shared by all subword transforms. "  # noqa: E501
+            "Including options for indicate subword model path, "
+            "`Subword Regularization <https://arxiv.org/abs/1804.10959>`_"
+            "/`BPE-Dropout <https://arxiv.org/abs/1910.13267>`_, "
```

Comment:
Is this kind of format handled natively or does it require an additional package?

Reply:
It's handled/supported in our setup, I find this syntax in the document of [sphinx-rtd-theme](https://sphinx-rtd-theme.readthedocs.io/en/stable/demo/demo.html#inline-markup).